src/main/java/org/kitteh/irc/client/library/element/ServerMessage.java

```diff
@@ -71,4 +74,31 @@
      */
     @Nonnull
     List<MessageTag> getTags();
+
+    /**
+     * Gets the named tag if present
+     *
+     * @param name tag name
+     * @return tag if present
+     */
+    default Optional<MessageTag> getTag(@Nonnull String name) {
+        return this.getTags().stream().filter(tag -> tag.getName().equals(Sanity.nullCheck(name, "Name cannot be null"))).findAny();
```

Comment:
Should do `Sanity.nullCheck(name, "Name cannot be null")` before this line, it only needs to be done once.